CONTRIBUTING.md

```diff
@@ -0,0 +1,32 @@
+# MobX Contributor Guide
+
+Welcome to a community of developers just like you, striving to create the best experience around MobX. We welcome anyone who wants to contribute or provide constructive feedback, no matter the age or level of experience.
+
+Here are some ways to contribute to the project, from easiest to most difficult:
+
+-   [Reporting bugs](#reporting-bugs)
+-   [Improving the documentation](#improving-the-documentation)
+-   [Responding to issues](#responding-to-issues)
+-   [Small bug fixes](#small-bug-fixes)
+
+## Issues
+
+### Reporting bugs
+
+If you encounter a bug, please file an issue on GitHub via the repository of the sub-project you think contains the bug. If an issue you have is already reported, please add additional information or add a 👍 reaction to indicate your agreement.
+
+Include in the issue a link to your reproduction. A couple good options are a small Github repo or a [CodeSandbox](https://codesandbox.io/s/minimal-mobx-react-project-ppgml).
+
+If you have a more complicated issue where it is helpful to run it locally, you can download CodeSandbox template and work on it and then commit into your GitHub repo.
+
+### Improving the documentation
+
+Improving the documentation, examples, and other open source content can be the easiest way to contribute to the library. If you see a piece of content that can be better, open a PR with an improvement, no matter how small! If you would like to suggest a big change or major rewrite, we’d love to hear your ideas but please open an issue for discussion before writing the PR.
+
+### Responding to issues
+
+In addition to reporting issues, a great way to contribute to Apollo is to respond to other peoples' issues and try to identify the problem or help them work around it. If you’re interested in taking a more active role in this process, please go ahead and respond to issues.
```

Comment:
Apollo

Reply:
Busted :) Yea, I borrowed it from there 😎 Thanks for the catch.